lib/recording/RecordingManager.js

```diff
@@ -182,7 +181,15 @@ class RecordingManager {
         let durationMSec =  recordingEntrySession.getDurationMSec();
         if ( durationMSec >= this.recordingMaxDurationInMs && !recordingEntrySession.disableRecording) {
             logger.info(`recording reached max duration [${mathUtils.durationToString(durationMSec, 2, mathUtils.HOUR_IN_MSEC)}] hrs`);
-            this.splitRecording(entryId, 'recording  reached max duration');
+            recordingEntrySession.is24By7RecordingEnabled()
```

Comment:
rename method to the correct feature name

Reply:
You did review before I did commit to changes... ;)